nextcord/channel.py

```diff
@@ -2285,6 +2301,13 @@ async def edit(self, *, reason=None, **options):
             The camera video quality for the stage channel's participants.
 
             .. versionadded:: 2.0
+        user_limit: :class:`int`
+            The maximum number of users allowed in the stage channel.
+
+            This must be between ``0`` and ``10,000``. A value of ``0`` indicates
+            no limit.
```

Comment:
This documentation is inconsistent with the definition in `abc`, is this intended?

Reply:
What documentation? `user_limit` is nowhere to be found in abc.py.

Reply:
Yep your right, misread the file. It's earlier in channel.py I meant

Reply:
I assume you mean VoiceChannel then? Yes it is different.